src/platform/bouffalolab/BL702L/BUILD.gn

```diff
@@ -60,6 +57,19 @@ static_library("BL702L") {
       "../common/BLEManagerImpl.h",
     ]
   }
+
+  if (chip_enable_factory_data || chip_enable_factory_data_test) {
+    if (chip_enable_factory_data) {
+      defines = [ "CONFIG_BOUFFALOLAB_FACTORY_DATA_ENABLE" ]
+    }
+    common_sources += [
+      "${chip_root}/src/credentials/CHIPCert.h",
```

Comment:
same comment as in the other places:
  - drop common_sources
  - use build header
  - you are not allowed to reach into files owned by other packages

Reply:
- common_sources removed
- Add source set `credentials_header` in `src/credentials/BUILD.gn` to expose used headers, and deps with this.